style/modules/grid.css

```diff
@@ -0,0 +1,20 @@
+.gridContainer{ /* a container with grid */
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-gap: 10px;
+    background: #F5F5F5;
+    padding: 10px;
+}
+
+.gridContainer * {  /* applies to all the items in the grid */
+    background: #508EFF;
+    border-radius: 15px;
+    margin: 10px;
+}
+/* column control */
+.gridCol-2 {grid-template-columns: repeat(2, 1fr);}
+.gridCol-3 {grid-template-columns: repeat(3, 1fr);}
+.gridCol-4 {grid-template-columns: repeat(4, 1fr);}
+.gridCol-5 {grid-template-columns: repeat(5, 1fr);}
+.gridCol-6 {grid-template-columns: repeat(6, 1fr);}
+
```

Comment:
one too many newlines here

Reply:
do you know a way I could simplify the grid column thingy? Like, is there a way to pass parameters through HTML so that I wouldn't have to make a seperate style for each column size? 
Hope this makes sense.

Reply:
If I understand right, you want something like `grid-template-columns: repeat(x, 1fr)`, where the `x` gets passed in? Nah, you can't do that with CSS 😛 

But this approach for "subclassing" is fine, and AFAIK it's also what the various CSS frameworks use. Keep in mind too that we'll only need the 2-column and 3-column versions (I think).